tests/test_integration.py

```diff
@@ -62,6 +62,81 @@ def prepare_temp_dir(doc_id: str, incremental_update_dir: str, limit):
 @patch.object(config, "DEVELOPMENT_MODE", new="true")
 @pytest.mark.usefixtures("cleanup_test_vespa_before", "cleanup_test_vespa_after")
 def test_integration(test_vespa):
+    """Run a single indexing and ensure all fields are populated"""
+    runner = CliRunner()
+    s3_fixture_dir = str(FIXTURE_DIR / "s3_files")
+
+    family_documents = [
+        "CCLW.executive.10014.4470",
+        "CCLW.executive.10002.4495",
+        "CCLW.document.i00000004.n0000",
+    ]
+
+    result = runner.invoke(
+        run_as_cli,
+        args=[
+            s3_fixture_dir,
+            "--index-type",
+            "vespa",
+        ],
+    )
+    assert result.exit_code == 0, (
+        f"Exception: {result.exception if result.exception else None}\n"
+        f"Stdout: {result.stdout}"
+        f"Trace: {traceback.print_exception(*result.exc_info)}"
+    )
+
+    for doc_id in family_documents:
+        vespa_data = get_vespa_data(test_vespa, FAMILY_DOCUMENT_SCHEMA, doc_id)
+        fixture_path = FIXTURE_DIR / "s3_files" / f"{doc_id}.json"
+        embeddings_path = FIXTURE_DIR / "s3_files" / f"{doc_id}.npy"
+        s3_data = ParserOutput.model_validate_json(fixture_path.read_text())
+        embeddings = np.load(embeddings_path)
+
+        vf = vespa_data["fields"]
+        assert vf["family_name"] == s3_data.document_name
+        assert vf["family_name_index"] == s3_data.document_name
+        assert vf["family_description"] == s3_data.document_description
+        assert vf["family_description_index"] == s3_data.document_description
```

Comment:
**question:** Is it correct that these should be the same value?

Reply:
Huh, thats interesting. Good spot... I wouldnt think so, but I also think thats what we have been loading into vespa for some time 🙀

Reply:
_oh no_

Reply:
Interestingly I'm not finding a sensible alternative value in the data for this field. It looks like its stored differently inv vespa, so maybe there is some reason for it... Time to descend into the depths of git history 🤿

Reply:
[Its an old one!](https://github.com/climatepolicyradar/navigator-search-indexer/pull/63/files#diff-b0feaf64dc00618448a877148f340b7c17ff15e36d8d69bd6519f427389aedf5R142-R143)

Reply:
So it's always been putting the same value in there. That seems odd!

Reply:
Okay, looks to me like it is not a bug after all... it is a bit ambiguous about the exact purpose, but afaict, one is used in ranking, the other is used in searching. Which is maybe a bit odd, but doesnt look broken at least 👍 good to check!

Reply:
Sounds good! Thanks for digging into it.